### ISSUE TITLE
Is this project still alive - unfixed critical issues with versions of used packages

### ISSUE THREAD

**Issue by BePo65** [indexzero/ps-tree#60]
Running `npm audit` lists many critical errors.

Since there are pull requests still open fixing some of these issues dating from 2022, it seems that this project is abandoned.

@indexzero : is it worth creating a fix for these issues? I would do this, only if we have a chance to get this project up to date on npmjs too.

**Comment by MikeMcC399** [comment]
@nelsonic 

Please clarify if you are accepting PRs for this repo. In a reply to https://github.com/indexzero/ps-tree/issues/61 you wrote that you can merge PRs, so it sounds like you have write privileges to the repo. Is that correct?

On the other hand there is a queue of PRs which have not been merged and the last merge into the `master` branch was https://github.com/indexzero/ps-tree/commit/2b6476c837bbbd63d1a80fa8cd06adde9f4c745b in November 2018 which is now 6 years ago, so it does not look as though there is any readiness to do routine maintenance to the repo.

**Comment by nelsonic** [comment]
@MikeMcC399, I was granted write access back in the day. ✍️ 
So can assist with maintenance. 🧹 
Starting with the CI side-quest: https://github.com/indexzero/ps-tree/issues/61

**Comment by MikeMcC399** [comment]
As discussed elsewhere, the last release was https://github.com/indexzero/ps-tree/releases/tag/1.2.0 in Nov 2018, so the answer is effectively that the project is no longer maintained if it does not have the ability to trigger new releases.

See also the npm registry version list https://www.npmjs.com/package/ps-tree?activeTab=versions

**Comment by indexzero** [comment]
This project is not dead. It's simply that – as John Lennon would say – _"Open Source is what happens when you're busy making other plans."_

Contributions are always welcome, but please forgive delays in my response. I started this project when I was single. I now have three kids. 

I get many emails for Github issues which makes it hard to parse out the signal from the noise. Recently, I created a new email address for `npm` only which is `npm@charlie.dev`. If you need my attention that's the best place to get it <3

**Comment by MikeMcC399** [comment]
@indexzero 

Welcome back!

- https://github.com/indexzero/ps-tree/issues/58 would be the critical issue to resolve and to release a fix for

**Comment by MikeMcC399** [comment]
Despite the post of @indexzero last month, there has been no objective change in the status of this npm package. The latest release on the [npm registry](https://www.npmjs.com/package/ps-tree?activeTab=versions) is still `1.2.0`, published on Nov 26, 2018, which the npm UI displays as being 7 years ago. 

The last merge to the `master` branch was https://github.com/indexzero/ps-tree/commit/635e1c3808f3df51308efbe080c1e5112c38a44c based on a PR that I provided in December 2024.

This does not look like an "active" project would look.

**Comment by tylercritchlow** [comment]
> Despite the post of [@indexzero](https://github.com/indexzero) last month, there has been no objective change in the status of this npm package. The latest release on the [npm registry](https://www.npmjs.com/package/ps-tree?activeTab=versions) is still `1.2.0`, published on Nov 26, 2018, which the npm UI displays as being 7 years ago.
> 
> The last merge to the `master` branch was [635e1c3](https://github.com/indexzero/ps-tree/commit/635e1c3808f3df51308efbe080c1e5112c38a44c) based on a PR that I provided in December 2024.
> 
> This does not look like an "active" project would look.

Did you ever find an alternative to this project that is active?

**Comment by MikeMcC399** [comment]
@tylercritchlow 

> Did you ever find an alternative to this project that is active?

- Another user proposed https://www.npmjs.com/package/tree-kill in https://github.com/bahmutov/start-server-and-test/issues/398#issuecomment-3131471319 as a replacement. I don't personally have any experience using `tree-kill` though.

**Comment by MikeMcC399** [comment]
> * Another user proposed https://www.npmjs.com/package/tree-kill in [Error: spawn wmic.exe ENOENT on current Windows systems bahmutov/start-server-and-test#398 (comment)](https://github.com/bahmutov/start-server-and-test/issues/398#issuecomment-3131471319) as a replacement. I don't personally have any experience using `tree-kill` though.

This was implemented in https://github.com/bahmutov/start-server-and-test which no longer depends on `ps-tree,` since the release [start-server-and-test@3.0.0](https://github.com/bahmutov/start-server-and-test/releases/tag/v3.0.0).
